execute/plugin.go

```diff
@@ -70,7 +70,7 @@ func NewPlugin(
 		syncFrequency(cfg.SyncFrequency),
 	)
 	if err := readerSyncer.Start(context.Background()); err != nil {
-		lggr.Errorw("error starting background reader syncer", "err", err)
+		lggr.Warnw("error starting background reader syncer", "err", err)
```

Comment:
@dimkouv should this just return an error? Plugin won't function correctly if the syncer isn't working right?